src/Rule/Url.php

```diff
@@ -80,29 +93,45 @@ public function getName(): string
         return 'url';
     }
 
+    /**
+     * @return string Ready to use regular expression pattern used for URL validation.
+     */
     public function getPattern(): string
     {
         return str_replace('{schemes}', '((?i)' . implode('|', $this->validSchemes) . ')', $this->pattern);
     }
 
     /**
-     * @return string[]
+     * @return string[] List of URI schemes which should be considered valid. By default, http and https
+     * are considered to be valid schemes.
      */
     public function getValidSchemes(): array
     {
         return $this->validSchemes;
     }
 
-    public function isEnableIDN(): bool
+    /**
+     * @return bool Whether validation process should take into account IDN (internationalized
+     * domain names). `false` meaning that validation of URLs containing IDN will always
+     * fail. Note that in order to use IDN validation you have to install and enable `intl` PHP
+     * extension, otherwise an exception would be thrown.
+     */
+    public function isIDNEnabled(): bool
```

Comment:
```suggestion
    public function isIdnEnabled(): bool
```

Reply:
Why? The property is caps-ed as well as original method.

Reply:
Usually in camelCase style acronyms used in lowercase also. For example, `URL` used as `url` almost always.

Reply:
We need that in the code style docs, I guess. And fixed everywhere.